CITATION.cff

```diff
@@ -0,0 +1,113 @@
+# This file is a citation metadata file in the Citation File Format
+# (http://citation-file-format.github.io/).
+#
+# It provides citation metadata for this version of stella
+
+cff-version: 1.1.0
+message: If you use stella in your work, please cite it using the following metadata.
+title: stella
+authors:
+  - family-names: Barnes
+    given-names: Michael
+    affiliation: University of Oxford
+    orcid: https://orcid.org/0000-0002-0177-1689
+
+  - family-names: St-Onge
+    given-names: Denis
+    affiliation: University of Oxford
+    orcid: https://orcid.org/0000-0003-3112-9221
+
```

Comment:
suggest adding Alexander von Boetticher, affiliated with University of Oxford and Robert Davies at York, as well as Jose Manuel Garcia-Regana at CIEMAT.  There may be others as well, but we can sort those out as they commit

Reply:
Done